pysteps/tests/test_extrapolation_semilagrangian.py

```diff
@@ -19,3 +19,12 @@ def test_semilagrangian():
     # result
     result = extrapolate(precip, velocity, num_timesteps)
     assert_array_almost_equal(result, expected)
+
+def test_ascending_time_step():
```

Comment:
From the name of the function is not clear what is being tested. 

What about: `test_not_ascending_timesteps_raise_exceptions`